user_class/spec/user_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative '../user'
+require 'factories'
+
+RSpec.describe User do
+  subject(:users) { described_class.new }
+
+  let(:info_user) { attributes_for(:user) }
+
+  describe '.create' do
+    context 'when all arguments are passed' do
+      it 'add an user' do
```

Comment:
Always third person.
```suggestion
      it "adds a user" do
```